maxim/logger/bedrock/async_client.py

```
@@ -39,6 +52,25 @@ async def converse_stream(
         additionalModelRequestFields: Optional[Dict[str, Any]] = None,
         **kwargs: Any,
     ) -> AsyncGenerator[BedrockStreamEvent, None]:
+        """Converse with the Bedrock client.
+
+        Args:
+            messages (Iterable[BedrockMessageParam]): The messages to send to the Bedrock client.
+            system (Optional[Union[str, List[Dict[str, str]]]]): The system message to send to the Bedrock client.
```

Pylint (3.3.7)

[convention] Line too long (114/100)

(C0301)